system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiCommandLineClientTest.java

```diff
@@ -54,15 +55,35 @@ void listParticipants() throws Exception {
 
         assertThat(getParticipants(cmd)).noneSatisfy(p -> assertThat(p.getUrl()).isEqualTo(idsUrl));
 
-        var addCmdExitCode = cmd.execute(
-                "-c", CLIENT_DID_WEB,
+        addParticipant(cmd);
+        assertThat(getParticipants(cmd)).anySatisfy(p -> assertThat(p.getUrl()).isEqualTo(idsUrl));
+    }
+
+    @Test
+    void participantStatus() throws Exception {
+        CommandLine cmd = RegistrationServiceCli.getCommandLine();
+
+        addParticipant(cmd);
+
+        var result = getParticipantStatus(cmd);
+        assertThat(result).isNotNull();
+        assertThat(result.getDid()).isEqualTo(CLIENT_DID_WEB);
+        assertThat(result.getUrl()).isEqualTo(idsUrl);
+        assertThat(result.getStatus()).isNotNull();
+    }
+
+    @Test
+    void participantStatus_notFound() {
+        CommandLine cmd = RegistrationServiceCli.getCommandLine();
+
+        var statusCmdExitCode = cmd.execute(
+                "-c", UNREGISTERED_CLIENT_DID_WEB,
                 "-d", DATASPACE_DID_WEB,
                 "-k", privateKeyFile.toString(),
                 "--http-scheme",
-                "participants", "add",
-                "--ids-url", idsUrl);
-        assertThat(addCmdExitCode).isEqualTo(0);
-        assertThat(getParticipants(cmd)).anySatisfy(p -> assertThat(p.getUrl()).isEqualTo(idsUrl));
+                "participants", "status");
+
+        assertThat(statusCmdExitCode).isEqualTo(1);
```

Comment:
should we also assert that stdout is empty?

Reply:
Sure, added. Thanks.